[16.0][MIG] mail_layout_preview: Migrate to version 16.0

sonhd91 opened the pull request.

pedrobaeza committed:
[ADD] mass_mailing_resend (#210)

* [ADD] mass_mailing_resend

Resend mass mailings
====================

A frequent need for users of mass mailings is to resend one mailing that has
already been sent in the past to new recipients that haven't received yet that
mail. But the problem is to know which are the applicable ones.

Odoo already includes a method in its mass mailing logic that avoids to resend
the same mail 2 times for one mass mailing, and for v9, there was a trick that
allows to modify the state of a mass mailing from kanban view, covering the
need.

But now on v10 both status bar in form view and dragging between states in
kanban are not allowed.

This module introduces a button to restart a mass mailing to draft state,
allowing you to reevaluate the sending domain or list for performing again
the mailing.

Usage
=====

* Go to *Mass mailing > Mailings > Mass Mailings*.
* Click on one record that is done or create a new one and send it.
* You will see a button called "Resend".
* If you click on it, mass mailing will be set to Draft again.

Known issues / Roadmap
======================

* Add an indicator / filter for knowing resent mailings.
* Include information on the number of new recipients to be sent on the
  resending (through `get_remaining_recipients` method).

pedrobaeza committed:
[MIG] mass_mailing_resend: Migration to 11.0

* Standard procedure
* Adopt README by fragments
* Adapt tests

ernestotejeda committed:
[MIG] mass_mailing_resend: Migration to 12.0

ernestotejeda committed:
[12.0][FIX] mass_mailing_resend: Remove alert-* class

simahawk committed:
Add mail_layout_preview

pedrobaeza committed:
[IMP] : black, isort

pedrobaeza committed:
[MIG] mass_mailing_resend: Migration to 13.0

Commit:
[IMP] pre-commit run -a

Commit:
[IMP] pre-commit run -a

phuctranfxvn committed:
[MIG] mail_layout_preview: Migration to 14.0

victoralmau committed:
[MIG] mass_mailing_resend: Migration to 14.0

[UPD] Update mass_mailing_resend.pot

[UPD] README.rst

Commit:
[FIX] mail_layout_preview: fix unit test

victoralmau committed:
[MIG] mass_mailing_resend: Migration to 15.0

TT40296

Commit:
[UPD] Update mass_mailing_resend.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-15.0/social-15.0-mass_mailing_resend
Translate-URL: https://translation.odoo-community.org/projects/social-15-0/social-15-0-mass_mailing_resend/

yostashiro committed:
[FIX] Move smtp info update logic to _prepare_email_message()

Since 15.0, smtp info is prepared in _prepare_email_message() which is called
from send_email(), and the smtp info manipulation should be done in this method
instead of in send_email().

bizzappdev committed:
[MIG][15.0]mail_layout_preview:Migration to 15.0

Commit:
[IMP] mass_mailing_resend: black, isort, prettier

Commit:
[MIG] mass_mailing_resend: Migration to 16.0

Commit:
[IMP] mail_layout_preview: black, isort, prettier

Commit:
[16.0][MIG] mail_layout_preview: Migrate to version 16.0

dzungtran89 commented:
it would be nice if you can review this PR https://github.com/OCA/social/pull/1085

OCA-git-bot committed:
Merge PR #1108 into 16.0

Signed-off-by hbrunn

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mail_outbound_static 16.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #1100 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update mass_mailing_resend.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-16.0/social-16.0-mass_mailing_resend
Translate-URL: https://translation.odoo-community.org/projects/social-16-0/social-16-0-mass_mailing_resend/

Commit:
[FIX] fix case when template doesn't have record to render preview
